src/app.ts

```diff
@@ -1,6 +1,6 @@
-const express = require("express");
-const swaggerUi = require("swagger-ui-express");
-const swaggerSpec = require("./swagger.js");
+import express from "express";
+import swaggerUi from "swagger-ui-express";
+import swaggerSpec from "./swagger.js";
 const app = express();
```

Comment:
Better left an empty line between imports and variables declarations or setup linter/formatter for doing such things automatically.

Reply:
Added empty lines

Reply:
Not exactly, just once between import block and other code like this:

```suggestion
import express from "express";
import swaggerUi from "swagger-ui-express";
import swaggerJSDoc from "swagger-jsdoc";

const app = express();
```

Reply:
Corrected